Fixes typo and improves definition of approach slab (#990)
buildingSMART/IFC4.3.x-development: docs/schemas/shared/IfcSharedBldgElements/Types/IfcSlabTypeEnum.md

The slab is used to represent a floor slab against the ground (and thereby being a part of the foundation). Another name is mat foundation.

### APPROACH_SLAB

> An approach slab is a reinforced or prestressed reinforced concrete slab providing transition from embankment to the bridge

Reviewer: ```suggestion
A slab used to provide a transition from embankment to the bridge.
```

Reviewer: I suggest to just deal with the typo and agree not to inject materials' info that may be perceived as constraint - we try to apply the same generic logic everywhere

Author: Not sure if it is a good idea to say "Part of the bridge abutment...", as this would imply that an approach slab belongs in the spatial structure of IfcBridgePart.ABUTMENT.

Perhaps a better description is:
The slab is used to provide a transition from embankment to the bridge.
